src/listener/starboard/starRemove.ts

```diff
@@ -0,0 +1,90 @@
+import { Listener } from 'discord-akairo';
+import { guildConfigs } from '../../guild/config/guildConfigs';
+import { MessageReaction, TextChannel, User, MessageEmbed } from 'discord.js';
+import { StarMessageModel } from '../../model/starboardMessage';
+import { findImageFromMessage } from '../../util/functions';
+
+export default class StarRemoveListener extends Listener {
+	constructor() {
+		super('starRemove', {
+			emitter: 'client',
+			event: 'messageReactionRemove',
+		});
+	}
+
+	async exec(reaction: MessageReaction, user: User) {
+		const msg = reaction.message;
+		if (!msg.guild) return;
+
+		if (
+			user.id === this.client.user?.id ||
+			msg.author.id === this.client.user?.id
+		)
+			return;
+
+		const config = guildConfigs.get(msg.guild.id);
+		const starboardConfig = config?.features.starboard;
+
+		if (
+			!config ||
+			!starboardConfig ||
+			!starboardConfig.channel ||
+			!starboardConfig.triggerAmount ||
+			(starboardConfig.ignoredChannels &&
+				starboardConfig.ignoredChannels.includes(msg.channel.id))
+		)
+			return;
+
+		const emojiAddedByUser = reaction.emoji.toString();
+		const starboardTriggerEmoji = starboardConfig.emoji ?? '⭐';
+
+		if (emojiAddedByUser !== starboardTriggerEmoji) return;
+
+		if (this.client.starboardCooldownManager.isOnCooldown(user.id)) return;
```

Comment:
Why add a cooldown when removing a star? In my opinion this should just be implemented when adding a star since if someone tries spam adding and removing a star, it would stop them when they try to add a star after removing their own star.